File: day3/Twosum.java
package day3;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        int [] new_nums = new int[2];
        int len = nums.length;

        for(int i = 0; i < len; i++){
            for(int j = i+1; j < len; j++){
                if((nums[i] + nums[j]) == target){
                    new_nums[0] = i;
                    new_nums[1] = j;
                }
            }

        }
        return new_nums;
    }
}

File: day3/twosum.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l = len(nums)
        new_nums = []
        for i in range(l):
            for j in range(i+1,l):
                if((nums[i] + nums[j]) == target):
                    new_nums.append(i)
                    new_nums.append(j)
        return new_nums
        